app.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+(function(){
+    angular
+    .module('TaskBastard', [])
+    .controller('TBController', function(){
+      var vm = this;
+      vm.tasks = data;
+      console.log(vm.tasks);
+
+      vm.create = function(task){
+        vm.tasks.push(task)
+        console.log(vm.tasks)
+      }
+
+      vm.destroy = function(task){
+      var index = vm.tasks.indexOf(task);
+      vm.tasks.splice(index, 1);
+      }
+
+
+  });
+})();
```

Comment:
You don't need `ngResource` to implement edit functionality. Let me know if you want to go over how to do that in this hard-coded scenario.